core/dotnet3.1/proxy/Apache.OpenWhisk.Runtime.Common/Init.cs

```diff
@@ -120,6 +121,17 @@ public async Task<Run> HandleRequest(HttpContext httpContext)
 
                 try
                 {
+                    // Export init arguments as environment variables
+                    if (message["env"] != null && message["env"].HasValues)
```

Comment:
@shawnallen85 this is to avoid null exceptions when the env is a "null" json value.
I opted to put the code here instead of earlier to let the quicker failures to come first.

Reply:
Sorry for the delay.

The code change looks good to me. Would it make sense to add it to the .NET Core 2.2 runtime as well?

Thanks!!
Shawn

Reply:
I didn't do it because of your note to the dev list to deprecate .NET 2.2.
Can add it to the PR if you think it's a good idea. I defer to your judgement.

Reply:
I figured that was the case but I wanted to make sure. I think it would be fine to keep .NET 2.2 as-is and move forward with making the change only in the 3.1 version of the code.